src/components/MultiSelect/MultiSelectDialog.tsx

```diff
@@ -113,121 +156,299 @@ export const MultiSelectDialog = chakra(
+                            : {
+                                isChecked: isChecked(id, item.id),
+                                onChange: onChange,
+                              })}
                         />
-                      )}
-                    </ListItem>
-                  ))}
-              </UnorderedList>
-              {isOpen && isLargerThanMobile && (
-                <ButtonGroup __css={styles.actionButtons}>
-                  <Button
-                    id={`ms-${id}-clear`}
-                    buttonType="link"
-                    type="button"
-                    onClick={onClear}
-                  >
-                    Clear
-                  </Button>
-                  <Button
-                    id={`ms-${id}-apply`}
-                    buttonType="primary"
-                    type="button"
-                    onClick={() => {
-                      // Close the multiselect on apply.
-                      setIsOpen(false);
-                      // Run the onApply prop function.
-                      onApply();
-                    }}
-                  >
-                    Apply
-                  </Button>
-                </ButtonGroup>
-              )}
+
+                        {item.children.map((childItem) => (
+                          <Checkbox
+                            key={childItem.id}
+                            marginInlineStart="0"
+                            __css={styles.menuChildren}
+                            id={childItem.id}
+                            labelText={childItem.name}
+                            name={childItem.name}
+                            isChecked={isChecked(id, childItem.id)}
+                            onChange={onChange}
+                            display={defaultChildCheckboxes}
+                          />
+                        ))}
+                      </>
+                    ) : (
+                      <Checkbox
+                        id={item.id}
+                        labelText={item.name}
+                        name={item.name}
+                        isChecked={isChecked(id, item.id)}
+                        onChange={onChange}
+                      />
+                    )}
+                  </React.Fragment>
+                ))}
+              </CheckboxGroup>
+              {showViewLabel()}
             </Box>
-          </FocusLock>
+          </>
+        );
+      };
+
+      const showViewLabel = () => {
+        if (itemsList.length > 0) {
+          return (
+            <Text
+              onClick={() => viewAllItems()}
+              __css={styles.viewAllButton}
+              id="text-btn"
+              marginTop="s"
+            >
+              {viewAllLabel}
+            </Text>
+          );
+        }
+      };
+
+      const itemsNotFound = () => {
+        if (itemsList.length === 0) {
+          return (
+            <Text id="text-btn" marginTop="xs" isItalic="true">
```

Comment:
`isItalic` expects a boolean value, not a string.
<img width="934" alt="Screen Shot 2023-12-12 at 12 35 10 PM" src="https://github.com/NYPL/nypl-design-system/assets/1280564/3c45e926-ba73-4e94-ba78-9f527a62072a">